build_che.sh

```diff
@@ -16,7 +16,9 @@ scl enable rh-maven33 'mvn -B clean install'
 cd ..
 
 git clone -b ${GIT_BRANCH} ${GIT_REPO}
-cd che 
+cd che
+# Use our modified stacks
+mv ./ide/che-core-ide-stacks/src/main/resources/stacks.json.centos \ 
+ ./ide/che-core-ide-stacks/src/main/resources/stacks.json
```

Comment:
Good point but we do not revert Dockerfile in the jenkins job https://github.com/kbsingh/build-run-che/blob/master/cico_build.sh#L28. That is done in rh-che repository and yes we need to update that too.

For consistency I would not revert stacks.json neither but I've changed the PR to use `cat` to do the replacement as it is done for the Dockerfile.